src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:18371/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:18371/project/issues?id=sample&branch=feature4&issues=AWEE9-XtweXi1q6dtQEE&open=AWEE9-XtweXi1q6dtQEE)

[ISSUE_KEY]: AWEE9-XtweXi1q6dtQEE